geminidr/f2/primitives_f2.py

```diff
@@ -25,8 +25,8 @@ class F2(Gemini, NearIR):
     tagset = {"GEMINI", "F2"}
 
     def __init__(self, adinputs, **kwargs):
-        super().__init__(adinputs, **kwargs)
         self.inst_lookups = 'geminidr.f2.lookups'
+        super().__init__(adinputs, **kwargs)
```

Comment:
Just curiosity, why did you move this line?

Reply:
I guess because the parent class' `__init__` will use it. 
Another way would be to make `inst_lookups` a class attribute.

Reply:
The standard MDFs for each instrument are retrieved via the calibration service, and therefore they need to be added when the databases are set up. The databases are set up in `PrimitivesBASE.__init__()` but the locations of the MDFs are derived from `self.inst_lookups`. Therefore we need to define `self.inst_lookups` _before_ calling `super().__init__()`.